### DISCUSSION TITLE
Unable to use the plugin

### DISCUSSION THREAD

**Discussion by mcanti** [MuiseDestiny/zotero-gpt#250] [Q&A]
Hi, I have zotero 7. Installed the plugin and set the secret key. 
When triggering any of the tags, the Progress window appears with the messages: 
Generating input content...
Characters XXX
Answering...

It then get's stuck, nothing happens after this. In the OpenAI portal I can see that the secret key has never been used. 
Can you help me out?

**Comment by zfb132** [comment]
Could you try another paper? Or can you share the paper?

**Reply by mcanti** [reply]
It does not seem to be dependent on the current selected item (if this is what you are referring to with paper). Any tag leads to the same behavior.
In the console I see this error appearing:
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/4506177/fd4e743b-8954-40e2-bf5e-e04e5c268031)

**Comment by Hua-jiu** [comment]
我也无法使用这个插件, 在配置好密钥后使用时无反应.
<img width="816" alt="image" src="https://github.com/MuiseDestiny/zotero-gpt/assets/78064555/79a02322-583b-4e45-b261-99d6d608fe00">

**Comment by HYH-SCU** [comment]
我也没法正常使用插件
添加插件后没有显示他的图标，问问题一直处于“回答·····”状态

**Reply by Jingxin-Zhang** [reply]
我也，怎么就我们这么少人有这个现象啊，都找不到解决办法
